include/deal.II/grid/grid_tools.h

```diff
@@ -643,6 +643,17 @@ namespace GridTools
   std::vector<std::set<typename Triangulation<dim,spacedim>::active_cell_iterator> >
   vertex_to_cell_map(const Triangulation<dim,spacedim> &triangulation);
 
+  /**
+   * Compute a global unique index for each vertex and hanging node associated
+   * to a locally owned active cell. The key of the map is the local index of
```

Comment:
global -> globally
associated to -> associated with

I think there is no need to mention hanging nodes -- to me, these are DoFs, not vertices, and it is clear that these points are included because they're vertices of some active cell.